docker/package/package_generator.py

```diff
@@ -143,6 +152,22 @@ def main():
                     package.source_archive = os.path.join(
                         args.sources_dir, source_archive
                     )
+                elif dl_sources:
+                    version = package.meta.version
+                    repo = "tezos-rc" if "rc" in version else "tezos"
+                    release = package.meta.release
+                    name = package.name
+                    url = f"https://launchpad.net/~serokell/+archive/ubuntu/{repo}/+sourcefiles/{name}/2:{version}-0ubuntu{release}~jammy/{name}_{version}.orig.tar.gz"
+                    try:
+                        urllib.request.urlretrieve(
+                            url,
+                            source_archive,
+                        )
```

Comment:
Maybe it would be nice to print a message here, saying that the source archive has been dowloaded.